setup.py

```diff
@@ -72,7 +72,7 @@
         'requests',
         'sphinx==1.6.7',
         'tox',
-        'wheel==0.30.0',
+        'wheel==0.34.2',
```

Comment:
I remember an [issue ](https://github.com/Azure/azure-cli/pull/12782)Azure CLI support pkginfo to read Azure CLI Extension's metadata instead of metadata.json, which means Azure CLI supported reading metadata from wheel built with both wheel==0.30.0 and wheel >= 0.31.0 from then, which is azure-cli-2.5.1.
(There is a breaking change that wheel>=0.31.0 no long support metadata.json anymore).

But extension haven't supported wheel >= 0.31.0 yet. So making it changed would cause user with older CLI version break if they install new azdev which is built from wheel==0.34.2 in this case considering there is 10000+ subscriptions using Azure CLI <2.5.1 still.

I think it might neet to be careful to upgrade this.